.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
This file & `__pycache__` should be in your `.gitignore` file. You should also add `venv` to your .gitignore file as well.